### ISSUE TITLE
Q3-FY23 Perform Quarterly Accessibility Audit (G&E)

### ISSUE THREAD

**Issue by jayna-SkylightDigital** [CDCgov/prime-simplereport#5493]
## **Objective** 
Maintain 508 compliance for an equitable user experience
## **Key Results (KRs)** (to be complete by end of June)
- Zero accessibility regressions since the summer 2022 audit in terms of overall application accessibility and 508 compliance.
## Additional context
In July 2022, the CDC Accessibility team reached out to SimpleReport via Index Analytics (Support) to alert us of a possible violation of Section 508 of the Rehabilitation Act. Our team was required to take immediate action to remedy the situation, performing an audit on the entire site and providing evidence that the offending issues had been addressed. Since that time, the SimpleReport team has performed quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

The **PRIME Accessibility Best Practices** can be found here: [https://cdc.sharepoint.com/:w:/r/teams/USDSatCDC/_layouts/15/Doc.aspx?sourcedoc=%7B31480032-AB2F-4106-A510-6AFA860003F7%7D&file=PRIME%20Accessibility%20Best%20Practices.docx&action=default&mobileredirect=true&DefaultItemOpen=1]

## Notes
- any issues relating to rule ID `reading-order-browse-outside-modal` for modals using react-modal can be ignored
- any issues regarding the default level of accessibility for the `<input type="date">` element can be ignored (unless someone using a different type of screen reader flags it as an issue -- [then we explore options in "future considerations"](https://github.com/CDCgov/prime-simplereport/issues/5862#issuecomment-1601311316))
